Intersect.Editor/Localization/Strings.cs

```diff
@@ -642,6 +642,8 @@ public partial struct AnimationEditor
 
             public static LocalizedString simulatedarkness = @"Simulate Darkness: {00}";
 
+            public static LocalizedString camerashake = @"Camera Shake (ms): {00}";
```

Comment:
Name this `CameraShake`, I don't expect you to change all of the other string names but I don't want to perpetuate the bad string names.

Reply:
For a reference: we need LocalizedString properties to follow the PascalCase naming convention 👍🏽 
Also, make sure to add the following JsonProperty:
`[JsonProperty(NullValueHandling = NullValueHandling.Ignore)]`